packages/@adobe/spectrum-css-temp/vars/spectrum-medium.css

```diff
@@ -231,10 +231,10 @@
   /* BEGIN RSP PATCH */
   /*
     This is not correct in DNA. Renamed from padding-y to padding-height to fix docs issue where fallback var replaced this value
-    (due to old spectrum-css postcss-custom-properties-custom-mapping plugin). Replaced var(--spectrum-global-dimension-static-size-75) with
-    static 6px value since a fallback isn't generated anymore due to the rename.
+    (due to old spectrum-css postcss-custom-properties-custom-mapping plugin). Replaced var(--spectrum-global-dimension-static-size-85) with
+    static 7px value since a fallback isn't generated anymore due to the rename.
   */
-  --spectrum-selectlist-option-padding-height: 6px;
+  --spectrum-selectlist-option-padding-height: 7px;
```

Comment:
We previously had `--spectrum-global-dimension-static-size-75` which apparently was overridden by a the plugin to the value of `--spectrum-global-dimension-static-size-85`... Checked spectrum-css and it should've been size-85, my bad.